main.js

```diff
@@ -0,0 +1,12 @@
+let linkInHeader = document.querySelector("header nav ul li");
+let spanEle = document.querySelector(".featured span");
+let lastPost = document.querySelector("main .posts article");
+let titles = document.querySelectorAll("main .posts h3");
+let titlesArr = Array.from(titles);
+
+linkInHeader.remove();
+spanEle.remove();
+lastPost.remove();
+for(let title of titlesArr) {
+    title.remove();
+}
```

Comment:
```suggestion
let linkInHeader = document.querySelector("header nav ul li");
linkInHeader.remove();

let spanEle = document.querySelector(".featured span");
spanEle.remove();

let lastPost = document.querySelector("main .posts article");
lastPost.remove();

let titles = document.querySelectorAll("main .posts h3");
for(let title of titles) {
    title.remove();
}
```
Nice job Lili! I reorganized your code for readability reasons. It's helpful to keep related code grouped together. You're selecting an element to delete it, so those two pieces of logic should be kept together. 

That being said, readability is subjective, so my opinion on this isn't necessarily the only correct way to do this. Just something to think about :)

Additionally, the step you had on line 5 `let titlesArr = Array.from(titles);` is not necessary. You can iterate over the `titles` node list without converting it into an array.

Reply:
Got it. Thank you so much, Sam!